article/_posts/2021-07-09-SigmaVAE.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: review
+title: "Simple and Effective VAE Training with Calibrated Decoders"
+tags: autoencoder VAE
+author: "Jon Haitz Legarreta Gorroño"
+cite:
+    authors: "Oleh Rybkin and Kostas Daniilidis and Sergey Levine"
+    title: "Simple and Effective VAE Training with Calibrated Decoders"
+    venue: "arXiv (rejected at ICLR 2021; accepted at ICML 2021)"
+pdf: "https://arxiv.org/pdf/2006.13202.pdf"
+---
+
+
+# Highlights
+
+- Their contributions are:
+ - A careful choice of the decoder distribution on naïve calibrated decoders
+often lead to worse results, can work very well, and removes the need to tune
+the additional parameter $$#beta$$.
```

Comment:
math error

Reply:
Sorry for the oversight. Fixed in 01620be. Reworked the sentence, since it was not complete.